recipe/conda_build_config.yaml

```diff
@@ -1,3 +1,3 @@
 # manual pins because we're linking migrations
 libprotobuf:
-  - 4.24.4
+  - 4.25.1
```

Comment:
Is it better to keep this an bump it or remove it?

Reply:
No, I should have updated it... 🤦